website/project/utils.py

```diff
@@ -1,16 +1,11 @@
 # -*- coding: utf-8 -*-
 """Various node-related utilities."""
-from website.project.views import node as node_views
-
 from website import mails
 from website import settings
 
 from website.util.permissions import ADMIN
 
-# Alias the project serializer
-serialize_node = node_views._view_project
```

Comment:
Why was this removed? 

This was added because the name and location (utils.py) make more sense than `_view_project`. `_view_project` still exists just for backwards compat.